index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio-Middle</title>
+</head>
+<body>
+    <section class="work">
+        <div class="work-title">
+            <h1 class="work-title-h1">My Recent Works</h1>
+            
+        </div>
+        <div class="work-items">
+            <hr>
+        </div>
+        <div class="work-items">
+            <img src="img/ImgPlaceholder.png" alt="">
+            <h2>
+                Multi-Post Stories
+            </h2>
+            <p class="work-items-p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+        </div>
+        <div class="work-items work-items-project">
+            <h5>Profesional Art Printing Data</h5>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="work-items-project-button">
+                <ul class="work-items-project-button-list">
+                    <li> <a href="#">Html</a> </li>
+                    <li> <a href="#">Booststrap</a> </li>
+                    <li> <a href="#">Ruby</a> </li>
+                    
+                </ul>
+            </div>
+        </div>
+        <div class="seeall">
+            <button>See Project</button>
+        </div>
+        <div class="work-items work-items-project">
+            <h5>Profesional Art Printing Data</h5>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="work-items-project-button">
+                <ul class="work-items-project-button-list">
+                    <li> <a href="#">Html</a> </li>
+                    <li> <a href="#">Booststrap</a> </li>
+                    <li> <a href="#">Ruby</a> </li>
+                    
+                </ul>
+            </div>
+        </div>
+        <div class="seeall">
+            <button>See Project</button>
```

Comment:
- [ ] All the buttons change of style in hover and active state.
![Captura de pantalla_2021-09-18_02-04-08](https://user-images.githubusercontent.com/53111464/133880125-49f93032-59fd-4682-8499-6467e1d1282f.png)